cmd/postgres_replicator/README.md

```diff
@@ -0,0 +1,66 @@
+# Postgres replicator
+
+This tool replicates streaming data from BigQuery into PostgreSQL.
```

Comment:
Maybe the word doesn't fit here, but I meant that the Postgres replicator exclusively operates
on data with stream-like qualities:
1. The data is sequential. For example, rows contain a timestamp column storing when they were added to the database. This is key for differentiating whether or not data should already exist in Postgres.
2. The data becomes slowly available over time. I was hoping by using the word "streaming" a user could make the inference that the Postgres replicator must exist as a long-running process, rather than a run-once and you're done kind of thing.

But like you said, it's all just data in BigQuery so maybe the word should be removed.

Reply:
IMHO "streaming" is usually used to describe the situation when you're getting realtime updates with new data. This is not our case so let's use different wording.